en/concept/flight_tasks.md

```diff
@@ -118,9 +123,10 @@ The instructions below might be used to create a task named _MyTask_:
       * @value 0 Direct velocity
       * @value 3 Smoothed velocity
       * @value 4 Acceleration based
+      * @value 5 Acceleration based
```

Comment:
You'd probably want to call it something like `My Task` for the drop down menu in the UI to select

Reply:
Let's just do this.

```suggestion
      * @value 5 My task
```